docs/team/charleslee01.md

```diff
@@ -11,28 +11,41 @@ command box provided.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Edit Command
-  * Implemented the EditEntityCommand, EditOwnerCommand and EditDogCommand that edits owners and dogs in the database.
-  * The EditOwnerCommand and EditDogCommand class are implemented such that it extends from the EditEntityCommand.
-  * In the process, I had to build `EditEntityCommandParser`, `EditOwnerCommandParser` and `EditDogCommandParser`to allow the proper functioning of this command.
+* **Code Refactoring**: Code Quality Improvements
+  * Increase defensive coding, exceptions and assertions.
+  * Update Javadoc comments so that it fits the current context.
```

Comment:
Can you rephrase this portion, seems like you are patch-writing from my PPP. And make sure PPP reflects your unique contributions, and that you have commits to back up what you write here.